modules/effects/src/index.ts

```diff
@@ -8,7 +8,10 @@ export { Actions, ofType } from './actions';
 export { EffectsModule } from './effects_module';
 export { EffectSources } from './effect_sources';
 export { EffectNotification } from './effect_notification';
-export { ROOT_EFFECTS_INIT } from './effects_root_module';
+export {
+  ROOT_EFFECTS_INIT,
+  rootEffectsInitAction,
```

Comment:
The action creator is created as `rootEffectsInit ` in `effects_root_module.ts`

```suggestion
  rootEffectsInit,
```